sdk/schemaregistry/Azure.Data.SchemaRegistry/samples/Sample01_HelloWorld.md

```diff
@@ -0,0 +1,122 @@
+# Registering and retrieving Avro, Json, and custom schemas (preview)
```

Comment:
Should we note this as a preview?

Reply:
Yes, I think that is a good idea.

Reply:
Though Avro should not be listed as preview.